services/slam/builtin/builtin.go

```diff
@@ -483,15 +483,9 @@ func NewBuiltIn(ctx context.Context, r robot.Robot, config config.Service, logge
 		dataRate = svcConfig.DataRateMs
 	}
 
-	var mapRate int
-	if svcConfig.MapRateSec <= 0 {
-		if svcConfig.MapRateSec == -1 {
-			mapRate = 0
-		} else {
-			mapRate = defaultMapRateSec
-		}
-	} else {
-		mapRate = svcConfig.MapRateSec
+	mapRate := defaultMapRateSec
+	if svcConfig.MapRateSec != nil && *svcConfig.MapRateSec >= 0 {
```

Comment:
[nit] log here if localization mode is enabled

Reply:
I was going to ask, do we want to also have logs stating if a default value is being used? Or are we alright with users having to look at the logs being sent by orbslam?

Reply:
Because we currently state that localization mode is active on the orbslam side already so interested to hear thoughts on double logging

Reply:
I think a debug log for defaults and an info log for localization, so a user should always be "aware" that they are in localization mode even if the slam algo logs are not enabled

Reply:
I do not think its a huge deal if both side have a log for enabling localization mode, that would help users know rdk and the slam algorithm are on the same page